browser/src/DataPage/exampleRawGraphQLQuery.ts

```diff
@@ -0,0 +1,126 @@
+const content = `# Lines beginning with a "#" character are comments that are ignored by the API.
+#
+# "VariantsInGene" is the operation name for the query. This is optional but
+# recommended.
+#
+query VariantsInGene {
+  # We could alternatively look up the gene by Ensembl ID by using the
+  # "gene_id" argument rather than "gene_symbol".
+  #
+  # The gene symbol is given with quotes, but not the reference genome or
+  # dataset. This is because the gene symbol is defined in the API schema as a
+  # string, but the reference genome is an enumerated type that can have only
+  # the value GRCh37 or GRCh38, and the dataset argument is constrained
+  # similarly. See
+  # https://graphql.org/learn/schema/#enumeration-types.
+  #
+  gene(gene_symbol: "BRCA1", reference_genome: GRCh38) {
+    variants(dataset: gnomad_r4) {
+      variant_id
+
+      # Position
+      pos
+
+      # dbSNP reference SNP IDs
+      rsids
+
+      transcript_id
+      transcript_version
+
+      # HGVS DNA, coding sequence, and protein consequences
+      hgvs
+      hgvsc
+      hgvsp
+
+      # VEP annotation
+      consequence
+
+      # Flags on the variant generally
+      flags
+
+      # The API separates the frequency data into exome data, genome data, and
+      # joint-frequency data. Which of these are available depends on the
```

Comment:
```suggestion
      # joint frequency data. Which of these are available depends on the
```
I don't think we need a hyphen here but optional